assets/package.json

```diff
@@ -7,22 +7,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "shresthagrawal.31@gmail.com",
+  "contributors": [
+    "prateek@marlin.pro"
+  ],
   "license": "ISC",
   "dependencies": {
-    "libp2p": "^0.25.4",
-    "libp2p-bootstrap": "^0.9.7",
-    "libp2p-kad-dht": "^0.15.0",
-    "libp2p-mdns": "^0.12.3",
-    "libp2p-secio": "^0.11.1",
-    "libp2p-tcp": "^0.13.0",
-    "libp2p-webrtc-star": "^0.16.1",
-    "libp2p-websockets": "^0.12.2",
-    "multiaddr": "^6.1.0",
-    "peer-info": "^0.15.1",
+    "it-pipe": "^1.1.0",
+    "libp2p": "^0.27.6",
+    "libp2p-bootstrap": "^0.10.4",
+    "libp2p-gossipsub": "^0.3.0",
+    "libp2p-kad-dht": "^0.18.6",
+    "libp2p-mdns": "^0.13.3",
+    "libp2p-mplex": "^0.9.5",
+    "libp2p-secio": "^0.12.4",
+    "libp2p-tcp": "^0.14.4",
+    "libp2p-webrtc-star": "^0.17.9",
+    "libp2p-websockets": "^0.13.6",
+    "multiaddr": "^7.4.3",
+    "peer-info": "^0.17.5",
```

Comment:
`pull-mplex` is missing?

Reply:
Shouldn't need it since it's been replaced by `it-pipe` right? I thought `pull-mplex` was part of the set of dependencies if you're working with pull-streams.